src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Figure implements Area {
+    private String color;
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String info() {
+        return null;
+    }

```

Comment:
make Figure implement interface `Drawable`, same as with AreaCalculator

Reply:
not fixed